collect_app/src/test/java/org/odk/collect/android/fragments/dialogs/ProgressDialogFragmentTest.java

```diff
@@ -106,6 +106,26 @@ public void setMessage_beforeDialogExists_setsMessageWhenDialogShown() {
         });
     }
 
+    @Test
+    public void setCancellable_makesTheDialogNotCancellable() {
+        activityScenario.onActivity(activity -> {
+            ProgressDialogFragment fragment = new ProgressDialogFragment();
+
+            Bundle args = new Bundle();
+            args.putBoolean(ProgressDialogFragment.CANCELABLE, false);
+            fragment.setArguments(args);
+            fragment.show(activity.getSupportFragmentManager(), "TAG");
+            shadowOf(getMainLooper()).idle();
+
+            AlertDialog dialog = (AlertDialog) fragment.getDialog();
+
+            activity.finish();
```

Comment:
I don't think this is right. Did you try commenting out your code to check this test passes and fails when you'd expect? 

Closing the activity might not "cancel" the dialog but it definitely wouldn't be on screen anymore so this test doesn't really make sense to me. You should be able to check whether the dialog is cancellable or not by calling `isCancellable` on it.